code/modules/roguetown/roguemachine/bounty.dm

```diff
@@ -104,17 +104,21 @@
 	var/list/eligible_players = list()
 
 	if(user.mind.known_people.len)
-		for(var/guys_name in user.mind.known_people)
-			eligible_players += guys_name
+		for(var/mob/living/carbon/human/H in world)
```

Comment:
there's GLOB.player_list or GLOB.alive_mob_list or perhaps some other list that might fit this better than an in world check.